tests/test_vectorstores.py

```diff
@@ -167,6 +174,42 @@ def test_firestore_add_vectors_assertions(
     )
 
 
+def test_firestore_add_image_vectors(
```

Comment:
Do you have time to add one more test for GCS inputs?

Reply:
I added a GCS url test